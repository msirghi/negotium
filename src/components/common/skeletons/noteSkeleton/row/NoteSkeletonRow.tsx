import { useNoteSkeletonStyles } from '../styles';
import { Row } from '../../../utilities/row/Row';
import { Grid, Skeleton } from '@mui/material';

const gridItemSizes = {
  xs: 12,
  sm: 12,
  md: 6,
  lg: 4,
} as any;

export const NoteSkeletonRow = () => {
  const classes = useNoteSkeletonStyles();
  return (
    <>
      <Grid item {...gridItemSizes}>
        <Skeleton className={classes.noteSkeleton} height={240} />
      </Grid>
      <Grid item {...gridItemSizes}>
        <Skeleton className={classes.noteSkeleton} height={240} />
      </Grid>
      <Grid item {...gridItemSizes}>
        <Skeleton className={classes.noteSkeleton} height={240} />
      </Grid>
    </>
  );
};
