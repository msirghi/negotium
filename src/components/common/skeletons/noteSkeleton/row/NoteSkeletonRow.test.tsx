import renderer from 'react-test-renderer';
import { NoteSkeletonRow } from './NoteSkeletonRow';

describe('NoteSkeletonRow', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<NoteSkeletonRow />);
    expect(tree).toMatchSnapshot();
  });
});
