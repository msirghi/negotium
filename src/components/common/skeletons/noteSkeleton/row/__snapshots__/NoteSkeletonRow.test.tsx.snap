// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteSkeletonRow should match the snapshot 1`] = `
Array [
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-157tyuy-MuiGrid-root"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse makeStyles-noteSkeleton-2 css-1n7gpw1-MuiSkeleton-root"
      style={
        Object {
          "height": 240,
          "width": undefined,
        }
      }
    />
  </div>,
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-157tyuy-MuiGrid-root"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse makeStyles-noteSkeleton-2 css-1n7gpw1-MuiSkeleton-root"
      style={
        Object {
          "height": 240,
          "width": undefined,
        }
      }
    />
  </div>,
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-6 MuiGrid-grid-lg-4 css-157tyuy-MuiGrid-root"
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse makeStyles-noteSkeleton-2 css-1n7gpw1-MuiSkeleton-root"
      style={
        Object {
          "height": 240,
          "width": undefined,
        }
      }
    />
  </div>,
]
`;
